deny.toml

```diff
@@ -20,6 +21,7 @@ allow = [
     "CC0-1.0",
     "ISC",
     "MIT",
+    "Unicode-3.0",
```

Comment:
https://spdx.org/licenses/Unicode-3.0.html reads fine to me. Seems the main thrust of the license is to be free to use as wished but not add the copyright holder in any advertising without consulting the copyright holder. What pulled this in and why?

Reply:
Thanks for taking time to look into this.

> what pulled this in?
It is through a transitive dependency on crates hosted in [icu4x](https://github.com/unicode-org/icu4x) - e.g [zerovec](https://github.com/unicode-org/icu4x/blob/main/utils/zerovec/LICENSE). Full list below,

Unicode-3.0 (19): icu_collections, icu_locid, icu_locid_transform, icu_locid_transform_data, icu_normalizer, icu_normalizer_data, icu_properties, icu_properties_data, icu_provider, icu_provider_macros, litemap, tinystr, writeable, yoke, yoke-derive, zerofrom, zerofrom-derive, zerovec, zerovec-derive


> why this is pulled in?

Upgrade to `object_store 0.11.1` seems to have pulled it in

```
│   ├── object_store v0.11.1
│   │   ├── reqwest v0.12.9
│   │   │   └── url v2.5.3
│   │   │       ├── form_urlencoded v1.2.1 (*)
│   │   │       ├── idna v1.0.3
│   │   │       │   ├── idna_adapter v1.2.0
│   │   │       │   │   │   ├── icu_collections v1.5.0
│   │   │       │   │   │   │   ├── displaydoc v0.2.5 (proc-macro) (*)
│   │   │       │   │   │   │   ├── yoke v0.7.4
│   │   │       │   │   │   │   ├── zerofrom v0.1.4 (*)
│   │   │       │   │   │   │   └── zerovec v0.10.4

```